.github/workflows/code-coverage-comment.yml

```diff
@@ -11,23 +11,6 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.event.workflow_run.event == 'pull_request_target' && github.event.workflow_run.conclusion == 'success' }}
     steps:
-      - name: Download Code Coverage
-        uses: actions/download-artifact@v4
```

Comment:
When deleting this, I went to double check why it wasn't working (given we just mucked with it).

That's when I found the:
```
    if: false # toggle this off, delete after 3/1/24 if nobody misses it
    # note: we split 'test_go' and 'test_coverage' steps because running race-detection + covprofile simultaneously is slow enough to cause flakes
```